infrastructure/variables.tf

```diff
@@ -61,3 +61,33 @@ variable "pgsql_sku" {
 variable "jenkins_AAD_objectId" {}
 
 variable "aks_subscription_id" {}
+
+variable "action_group_name" {
+  description = "The name of the Action Group to create."
+  type        = string
+  default     = "wa-support"
+}
+
+variable "email_address_key" {
+  description = "Email address key in azure Key Vault."
+  type        = string
+  default     = "db-alert-monitoring-email-address"
+}
+
+variable "cpu_threshold" {
+  default     = 80
```

Comment:
provide cpu_threshold, memory_threshold and storage_threshold only when configurations is different than 80

otherwise it will use default values provided in terraform-module-postgresql-flexible